app/views/abrs/_state_fields.html.haml

```diff
@@ -112,16 +112,34 @@
       }    
     });
   }
+
+  function supplementDisabledFields($) {
```

Comment:
Is this necessary? With radio/checkboxes a blank-valued hidden field gets overwritten by the real checkbox, so in theory wouldn't these work the same way?
Alternatively, can we just enable/disable the hidden fields instead of adding/removing from the dom?

Reply:
You're suggesting adding a hidden blank field for everything (read: all of the step3 custom fields) up front, and just leaving them there?  ... either client side, or server side....

Yes, that might work.  I approached this 1) to try and not change the back end ruby code, and 2) to do the minimal change possible. 

I know this works.  

I can test a different version which just puts them there (client-side) and leaves them there.

Reply:
Although... see below, your other idea seems to work more cleanly.